complete/_rg

```diff
@@ -270,6 +270,8 @@ _rg() {
     '*--ignore-file=[specify additional ignore file]:ignore file:_files'
     '(-v --invert-match)'{-v,--invert-match}'[invert matching]'
     '(-M --max-columns)'{-M+,--max-columns=}'[specify max length of lines to print]:number of bytes'
+    '--max-column-preview[show preview for long lines when limit is enabled]'
```

Comment:
These two should be made exclusive with each other, probably by breaking them out into a group with `+ '(max-column-preview)'` or similar

Also, i would suggest the description `show preview for long lines (with -M)` — that matches the wording for similar options, and might be a little clearer